src/components/screens/multiSignature/summary/summary.js

```diff
@@ -27,7 +27,7 @@ const Member = ({ member, i, t }) => (
   </div>
 );
 
-const Members = ({ members, t }) => {
+export const Members = ({ members, t }) => {
```

Comment:
Should we consider moving this to `shared` components? This is already being used at more than 1 places.

Reply:
I just didn't know if I'll cause a conflict with your other PRs.

Reply:
Done